crates/builder-api/src/v0_1/builder.rs

```diff
@@ -95,6 +106,7 @@ impl tide_disco::error::Error for Error {
             Error::TxnSubmit { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             Error::Custom { .. } => StatusCode::INTERNAL_SERVER_ERROR,
             Error::BuilderAddress { .. } => StatusCode::INTERNAL_SERVER_ERROR,
+            Error::TxnStatGet { .. } => StatusCode::BAD_REQUEST,
```

Comment:
Why is this a `BAD_REQUEST`? Not sure which one is the most suitable though, maybe `INTERNAL_SERVER_ERROR`, since similar to the `BuilderAddress` error we are failed to get some info here.

Reply:
Not sure neither. Previously I thought this is a request therefore setting, but it's true that it's similar `BuilderAddress` and `TxnSubmit`. Changed to `INTERNAL_SERVER_ERROR` to keep consistency.